geospacepy/sun.py

```diff
@@ -247,6 +249,9 @@ def local_hour_angle(jds,glons):
     #But the hour angle is defined as positive opposite the earth's rotation
     phi = np.radians(glons)
     lhas = (gmst+phi) - sra
+
+    # Wrap lhas to be between [-pi,pi]
```

Comment:
```suggestion
    # Wrap `lhas` to be between -pi and pi
```

At first I thought this was a typo, so added the accents marks to indicate it is referring to the variable below.